From f3e15f58f30731de882ecd2f6a625fbf8fcb7f14 Mon Sep 17 00:00:00 2001
From: Alexander Graf <agraf@suse.de>
Date: Thu, 8 Jun 2017 17:30:31 +0200
Subject: [PATCH 146/174] arm: Move voffset into .hyp.text
Patch-mainline: Never, only needed in 4.4 based kernel
References: bsc#1031492

Same as for arm64, let's move the voffset variable into .hyp.text so that
it gets covered by the idmap page table.

Signed-off-by: Alexander Graf <agraf@suse.de>

---
 arch/arm/mm/mmu.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mm/mmu.c b/arch/arm/mm/mmu.c
index d65cd0e..a5e9d18 100644
--- a/arch/arm/mm/mmu.c
+++ b/arch/arm/mm/mmu.c
@@ -87,7 +87,7 @@ struct cachepolicy {
 #define s2_policy(policy)	0
 #endif
 
-unsigned long kimage_voffset __ro_after_init;
+unsigned long kimage_voffset __section(.hyp.text);
 
 static struct cachepolicy cache_policies[] __initdata = {
 	{
-- 
1.8.5.6

